vera/README.md

```
@@ -125,7 +125,7 @@ execute(compile_result, ...)    # → run WASM via wasmtime
 | `codegen/` | 15,855 | Compile | Codegen orchestrator (mixin package) | `compile()`, `execute()` |
 | `  api.py` | 1,341 | | Public API, dataclasses, `compile()`/`execute()` orchestration, core IO host bindings (#421) | |
 | `  memory.py` | 105 | | Compile-time ADT layout helpers (`ConstructorLayout`, alignment) (#421) | |
-| `  core.py` | 1,960 | | CodeGenerator class, orchestration, ability op rewriting (Pass 1.6) | |
+| `  core.py` | 2,304 | | CodeGenerator class, orchestration, ability op rewriting (Pass 1.6), skip propagation to callers (#1100) | |
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)